topics/mahapps/index.md

```diff
@@ -0,0 +1,12 @@
+---
+aliases: mahapps-metro, metro
+created_by: Paul Jenkins
+display_name: "MahApps.Metro"
+github_url: https://github.com/MahApps/MahApps.Metro
+logo: mahapps.png
+released: January 29, 2011
+short_description: "A toolkit for creating metro-style WPF applications."
```

Comment:
can you remove the quotation marks here?

Reply:
@nayafia done